src/plugins/template/tests/functional/op_reference/base_reference_test.cpp

```diff
@@ -109,17 +109,31 @@ void CommonReferenceTest::ValidateBlobs(const ov::Tensor& refBlob,
         case ov::element::f16:
         case ov::element::f32:
         case ov::element::f64:
+        case ov::element::i4:
         case ov::element::i8:
         case ov::element::i16:
         case ov::element::i32:
         case ov::element::i64:
+        case ov::element::u1:
+        case ov::element::u3:
+        case ov::element::u4:
+        case ov::element::u6:
         case ov::element::u8:
         case ov::element::u16:
         case ov::element::u32:
         case ov::element::u64:
+        case ov::element::nf4:
+        case ov::element::f8e4m3:
+        case ov::element::f8e5m2:
```

Comment:
[ov::test::utils::compare](https://github.com/openvinotoolkit/openvino/blob/bcf8449dcf7bf2cf9e36d817c6ac0aa1d4cbd428/src/tests/test_utils/common_test_utils/src/ov_tensor_utils.cpp#L609-L656) doesn't support all of those newly added types above, that's why there is fall to next `switch` with legacy raw data comparison.
That's the reason (I presume) of skipping a few tests here - is it intentional?

Reply:
yes, it was intentional, but now I have removed tests from skips and add to code `if` to avoid comparison by non legacy way for such types and have created task to support that types